core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.controller.ts

```diff
@@ -61,6 +61,13 @@ angular.module('oppia').controller('AddMisconceptionModalController', [
     $scope.misconceptionMustBeAddressed = true;
     $scope.misconceptionNameIsDuplicate = false;
 
+    $scope.isEditorContentLengthExceeded = function() {
+      return (
+        $scope.misconceptionNotes.length > 1000 ||
+        $scope.misconceptionFeedback.length > 1000
```

Comment:
Almost! The validation check that we want is that the length of the **HTML string** is at most 1000 characters. See the other 2 PRs in issue #13764 that deal with HTML string length limits for samples. FYI: adding a single image to the content could add hundreds of characters to the HTML string even though an image isn't really "characters".

Reply:
Thanks for pointing that out . I will resolve that immediately !!

Reply:
hey @EricZLou , wanted to cross check some things before making a PR . Since i am capturing the whole html document inside the $scope.misconceptionNotes ( <p>Random Text</p> ) , it will also capture other things provided by skills-editor that you mentioned like images , collapsable inputs , links , formulas . So what i wanted to ask is that how would i take measure of the input chars / content length for all these other elements . Or i should just simply exclude these items from adding to overall input chars i.e. not taking them into consideration .

Reply:
Thanks for cross checking with me! I may have missed something here, and you may have been right originally. Add an image to the misconception note and feedback, and print out `$scope.misconceptionNotes` and `$scope.misconceptionFeedback` to see if they include `<img>` tags! If they do, then you actually don't need to change your code at all. We want to capture all of the normal characters as well as images, collapsible inputs, etc (once everything is converted to an HTML string).

Reply:
![html_capture](https://user-images.githubusercontent.com/63491234/141732269-b8541587-97cf-42e5-bebf-4fed41101b26.png)

@EricZLou  It is just as you said , we are capturing all the html document inserted by skill editor and not some image bytes data . So i don't think there is any need to change to code . However I will update other changes that you requested .

Reply:
Awesome -- thanks @ashish-patwal for checking this!!